### PR TITLE
Third Party Page

### PR DESCRIPTION
**Pull request by snshamwow**
Implementing a search feature using NewsAPI to find crypto news.

### CODE REVIEW

File: src/components/ThirdPartyEducation.js [hunk 1]
```diff
@@ -1,13 +1,73 @@
...
+        const newsList = document.querySelector('.news-list');
+
+        if (input.value === ''){
+            alert('Input field is empty.')
+            return
+        }
+      
+        newsList.innerHTML = ''
+      
+        const apiKey = '8577c03e48d246b3b0ca7329456e3e84'
```
Comment (IKwiotek) [inline src/components/ThirdPartyEducation.js]: The API key should not be included in source control. I mentioned this to other teams during the Sprint 2 presentation. 

Once the key is removed from source control, it should be rotated to ensure that the original value cannot be used maliciously.

**Review by IKwiotek — CHANGES_REQUESTED** [review]

**Review by IKwiotek — APPROVED** [review]
API Key is now removed, Approving PR.

**Comment by IKwiotek** [comment]
@vionaveseli I added you as an approver for this PR.

As discussed, we will need to save the API Key on our local machine and insert it into the TPE file before deploying. I'm not thrilled with this approach, but until we figure out how to pull the `newsapikey` environment variable from the back-end, this will suffice.

**Comment by IKwiotek** [comment]
We will want to deploy these changes to the hosted site prior to approving the PR as well.

**Review by vionaveseli — APPROVED** [review]
Approved. This should be one of the lasts merges we have in the main ensure other members don't run into problems when adding/running main or Sadeem should send the api key to each member so they can add it in on their local machine.

**Pull request merged**